lib/balance-tx/lib/internal/Internal/Cardano/Write/Tx/Balance.hs

```diff
@@ -637,7 +643,7 @@ balanceTransactionWithSelectionStrategyAndNoZeroAdaAdjustment
     genChange
     s
     selectionStrategy
-    ptx@(PartialTx partialTx inputUTxO redeemers)
+    ptx@(PartialTx partialTx inputUTxO redeemers nTimelockWits)
```

Comment:
The naming seems a bit inconsistent:
- `nTimelockWits` vs `timelockVkCounts` (terms)
- `SpecifiedTimelockScriptVkCounts` (type)

Suggestions:
- `TimelockScriptVkCounts` (type)
- `timelockScriptVkCounts` (value)

Reply:
Thanks! I'll go with

- `TimelockScriptVkCounts` (type)
- `timelockVkCounts` (value)

to save space

1666baed71623ab188535ace89b9613b7eb58fc6